template.yaml

```diff
@@ -299,6 +330,12 @@ Resources:
                 - kms:Decrypt
               Resource:
                 - !GetAtt QueueKmsKey.Arn
+            - Sid: DynamoDBPut
+              Effect: Allow
+              Action:
+                - dynamodb:PutItem
```

Comment:
Just wondering if we might need to query as well from the ProcessDataRequest function? Things like "is there already processing happening on this ticket"? We can add it later I guess though, so this is more of a point for discussion.

Reply:
It's possible, I'd rather add it if we definitely need it though.